recipe/meta.yaml

```diff
@@ -1,15 +1,18 @@
 {% set name = "ocl-icd" %}
-{% set version = "2.2.12" %}
-{% set sha256 = "17500e5788304eef5b52dbe784cec197bdae64e05eecf38317840d2d05484272" %}
+{% set version = "2.99_97e3f" %}
```

Comment:
@isuruf What do you think of this? (Particularly the version number? Also, is there a way to make this not the installed-by-default version?) Now that CL 3.0 is released, I'm trying to get support for it in shape. But to do so in PyOpenCL, I need the 3.0 headers.

x-ref: https://github.com/inducer/pyopencl/pull/376

Reply:
We can add publish this recipe to `conda-forge/label/ocl-icd-dev` channel. That way, it's not installed by default and the version can be `3.0.0.dev0` assuming `ocl-icd` is going to release `3.0.0` next.

Reply:
> We can publish this recipe to conda-forge/label/ocl-icd-dev channel.

That sounds like a good plan. How does that work?

> assuming `ocl-icd` is going to release `3.0.0` next.

Hard to know, but likely.

Reply:
Thanks!